From 997b07491fd9e34bf2707652a4694c17ffac9465 Mon Sep 17 00:00:00 2001
From: Guillem Jover <guillem@debian.org>
Date: Wed, 8 May 2013 11:44:22 +0200
Subject: [PATCH] dh_installinfo: Stop inserting dependencies for partial
 upgrades

This was introduced in 2009 to support partial upgrades from lenny
to squeeze, which are long time unneeded.
---
 dh_installinfo | 2 --
 1 file changed, 2 deletions(-)

diff --git a/dh_installinfo b/dh_installinfo
index c707a83..2c3d9e9 100755
--- a/dh_installinfo
+++ b/dh_installinfo
@@ -71,8 +71,6 @@ foreach my $package (@{$dh{DOPACKAGES}}) {
 		doit("cp",@info,"$tmp/usr/share/info");
 		doit("chmod","-R", "go=rX","$tmp/usr/share/info/");
 		doit("chmod","-R", "u+rw","$tmp/usr/share/info/");
-		# For transition, could be removed later.
-		addsubstvar($package, "misc:Depends", "dpkg (>= 1.15.4) | install-info");
 	}
 }
 
-- 
1.8.3.rc1

